osquery/tables/cloud/azure/azure_instance_metadata.cpp

```diff
@@ -0,0 +1,65 @@
+/**
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under both the Apache 2.0 license (found in the
+ *  LICENSE file in the root directory of this source tree) and the GPLv2 (found
+ *  in the COPYING file in the root directory of this source tree).
+ *  You may select, at your option, one of the above-listed licenses.
+ */
+
+#include <boost/property_tree/json_parser.hpp>
```

Comment:
Please use `<osquery/utils/json/json.h>`, the wrapper around RapidJSON, it is much faster.

Reply:
Will do -- I'll refactor to use the wrapper in a bit.